cmake/FindGflags.cmake

```diff
@@ -1,582 +0,0 @@
-# Ceres Solver - A fast non-linear least squares minimizer
```

Comment:
So glad to see that we can remove this very long source file!

Just for curiosity, why could we remove it? I noticed that in [this change](https://github.com/PaddlePaddle/Paddle/pull/4890/files#diff-a957dc4114d81bc0130e029e82c9e938L1) to the CMakeLists.txt file, we don't need to specify that a target depends on glog, but only on gflags. Is this related with the removal of this `FindGFlags.cmake` file?